pkg/dotc1z/c1file.go

```diff
@@ -21,10 +21,19 @@ type C1File struct {
 	outputFilePath string
 	dbFilePath     string
 	dbUpdated      bool
+	tempDir        string
+}
+
+type C1FOption func(*C1File)
+
+func WithC1FTmpDir(tempDir string) C1FOption {
+	return func(o *C1File) {
+		o.tempDir = tempDir
+	}
 }
 
 // Returns a C1File instance for the given db filepath.
-func NewC1File(ctx context.Context, dbFilePath string) (*C1File, error) {
+func NewC1File(ctx context.Context, dbFilePath string, opts ...C1FOption) (*C1File, error) {
```

Comment:
style nit: not sure why this can't just be `New()` instead of ` NewC1File`?

Reply:
This package also contains `NewC1ZFile()`, so I think it was to disambiguate. We could probably move the details of that function into an option for this function. Will consider that in a separate PR since it'd be API breaking.